src/cpp/rtps/builtin/discovery/endpoint/EDPServerListeners.hpp

```diff
@@ -72,6 +72,10 @@ class EDPServerPUBListener : public fastrtps::rtps::EDPBasePUBListener
             fastrtps::rtps::RTPSReader* reader,
             const fastrtps::rtps::CacheChange_t* const change) override;
 
+    void continue_with_writer(
+            fastrtps::rtps::RTPSReader* reader,
+            fastrtps::rtps::CacheChange_t* change);
```

Comment:
This API is intended to be called exclusively internally to the class. Please, make it private. The same applies to the EDPServerSUBListener

Reply:
made all new methods private in [391527d](https://github.com/eProsima/Fast-DDS/pull/4240/commits/391527df3729a32aadbeb5815d70fd172648d916)